CMakeLists.txt

```diff
@@ -53,6 +53,19 @@ if(LCM_ENABLE_JAVA)
 endif()
 
 # Python
+if(LCM_ENABLE_PYTHON AND APPLE)
+  find_program(PYTHON_CONFIG_EXECUTABLE python-config)
+  if (NOT PYTHON_CONFIG_EXECUTABLE)
+    message(SEND_ERROR "python-config executable not found, but python is required.")
```

Comment:
On second thought, what about unconditionally adding `python-config --prefix` to `CMAKE_PREFIX_PATH` if `python-config` can be found? (I.e. don't make looking for it conditional on anything.) Does that work?

...especially as the value you are using for `PYTHON_LIBRARY` is wrong on platforms with arch-dependent library paths (e.g. any modern Linux).